src/odemis/cli/main.py

```diff
@@ -1015,7 +1015,7 @@ def main(args):
                 dataflows = ["data"]
             else:
                 dataflows = options.acquire[1:]
-            filename = options.output.decode(sys.getfilesystemencoding())
+            filename = to_str_escape(options.output)
```

Comment:
You *cannot* escape the string. It's going to add `\` at every non-ascii character. What's wrong with the original code? Is it just because it doesn't work with Python 3? If so, I think you should do the same as (I've implemented) in the pylibtiff.open() code:
https://github.com/pearu/pylibtiff/blob/master/libtiff/libtiff_ctypes.py#L500

It sounds also like there should be a test case that use this code, to check it work. For example, request to save a file as `TÈßt.tiff", and check that a file named like this is created.

Reply:
* I updated test_acquire to use TÈßt.tiff as picture name instead of test.tiff
* for the filename I decode based on the whether it's unicode
```
            if isinstance(options.output, unicode):  # python3
                filename = options.output
            else:  # python2
                filename = options.output.decode(sys.getfilesystemencoding())
```